.circleci/config.yml

```diff
@@ -599,6 +671,7 @@ jobs:
 
 
 workflows:
+  version: 2
```

Comment:
q: interesting! I didn't know you could version CircleCI workflows. What would be the effect?

Reply:
It was used to issue warnings for deprecation or breaking changes. Although as per docs for v2.1 config we do not require this anymore.